spikesorters/spyking_circus/spyking_circus.py

```diff
@@ -91,7 +91,11 @@ def _setup_recording(self, recording, output_folder):
 
         # save binary file
         file_name = 'recording'
-        np.save(str(output_folder / file_name), recording.get_traces().astype('float32'))
+        #np.save(str(output_folder / file_name), recording.get_traces().astype('float32'))
+        raw_filename = output_folder / (file_name + '.raw')
+        n_chan = recording.get_num_channels()
+        chunk_size = 2 ** 24 // n_chan
+        recording.write_to_binary_dat_format(raw_filename, time_axis=0, dtype='float32', chunk_size=chunk_size)
```

Comment:
Oups. Finally not because the binary is copied inside the folder anyway.
Kave a look to KS2, KS, TDC where I put a trick to avoid this long copy.
But that the file must to unchanged!!!! So no overwrite=True.